source/index.html.md

```diff
@@ -19,223 +15,660 @@ search: true
 code_clipboard: true
 ---
 
-# Introduction
+# Zappi API
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+The Zappi API is REST-based and returns responses in JSON.
 
-We have language bindings in Shell, Ruby, Python, and JavaScript! You can view code examples in the dark area to the right, and you can switch the programming language of the examples with the tabs in the top right.
+To use our API, you’ll need to contact us in order to have a public integration created.
+
+# Beta
+
+Please note that as we are still finalizing our API, this spec is subject to change and should not be considered final.
+
+# Change Log
+
+## 16 August 2021
+
+Added ‘visibility’ to the order payload.
+
+## 19 July 2021
+
+Added `workspace_id` as an optional parameter on `POST /orders.`
+Added `workspace_id` to the order payload.
+
+## 14 July 2021
+
+Added `root_workspace_id` to `GET public_integrations/identity` payload.
+Added `GET /workspaces/{id}` endpoint.
+
+## 15 April 2021
+
+Updated `GET /Orders` rate limit from 10 to 20.
+Increased default pagination size from 30 to 100.
+
+## 8 April 2021
+
+Added `product_id` and `country_code` to the order payload.
+
+# API Overview
+
+Zappi allows 3rd parties to create public integrations on our platform. This will allow other companies and developers to build integrations which can be installed by different customer groups. All public integrations will need to be reviewed and approved by Zappi. Access tokens for public integrations will expire, and will need to be regenerated when necessary, using the client credentials oAuth flow. In order to make use of a public integration, a customer group will need to install it. Public integrations can be uninstalled at any point in time.
+Public Integrations
```

Comment:
Is this a sub heading? " Public Integrations"
Perhaps a new line and an h3 if it is

Reply:
wild, that is actually a copy paste error. Thanks. Will remove it because its in the following section.

🍝